bin/hummingbot_quickstart.py

```diff
@@ -61,9 +65,12 @@ def autofix_permissions(user_group_spec: str):
     project_home: str = os.path.realpath(os.path.join(__file__, "../../"))
     subprocess.run(f"cd '{project_home}' && "
                    f"sudo chown -R {user_group_spec} conf/ data/ logs/", capture_output=True, shell=True)
```

Comment:
Since you're sharing the certificates and gateway config path between containers, you should include them into autofix permissions as well.

Currently, when running hummingbot as Docker container, `gateway generate-certs` would fail because the certs directory permissions are still root.

Reply:
I probably missed that but I actually did some testing before pushing and didn't have any issues.